.github/workflows/pytest.yml

```diff
@@ -0,0 +1,26 @@
+name: Python Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  pyTest:
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+    runs-on: ubuntu-latest
```

Comment:
We might need to test on Bionic and Centos 7 because of PPI.

Reply:
This is just a starter, we can extend it. This is for testing against the relevant Python versions. I also think we should run against xenial and bionic with their default python version.